packages/material-ui-utils/src/useId.ts

```diff
@@ -8,7 +8,7 @@ export default function useId(idOverride?: string): string | undefined {
       // Fallback to this default id when possible.
       // Use the random value for client-side rendering only.
       // We can't use it server-side.
-      setDefaultId(`mui-${Math.round(Math.random() * 1e5)}`);
+      setDefaultId(`mui-${Math.round(Math.random() * 1e9)}`);
```

Comment:
Reducing the probability by one order of magnitude might have been enough

```suggestion
      setDefaultId(`mui-${Math.round(Math.random() * 1e6)}`);
```

but it 1e9 seems still not too noisy in the DOM, so why not:

<img width="425" alt="Capture d’écran 2021-06-15 à 11 48 43" src="https://user-images.githubusercontent.com/3165635/122032361-da572280-cdcf-11eb-862a-53a1ef234d5a.png">

Reply:
You want to tell me that 4 more bytes makes the difference when we have these class names?

Reply:
The opposite